js/shared_controls.js

```diff
@@ -936,6 +936,18 @@ function loadCustomList() {
 		});
 }
 
+function isUsingTemplates() {
+	return  (window.location.href.substring(window.location.href.lastIndexOf('/')).indexOf('.template.html') >= 0);
+}
+
+function get1vs1Page() {
```

Comment:
...this should really be done by the template compiler, not the code.

Like just update `./build` to find/replace `.template.html` with `.html`, then the rest of this code is unnecessary.

Reply:
But wasn't the whole point to not force people to run the build script

Reply:
Exactly!

Reply:
Specifically, you make the links to `.template.html`, so they work without the build script, and then the build script replaces `.template.html` with `.html`, so it works in the live version.

Which is what I _thought_ you were going to do until you dropped all these unnecessary functions. :p

Reply:
oohhhhh I get it now